src/TufValidatedComposerRepository.php

```diff
@@ -59,6 +60,11 @@ public function __construct(array $repoConfig, IOInterface $io, Config $config,
                 $metadataUrl .= '/';
             }
 
+            $maxBytes = $repoConfig['tuf']['max-bytes'] ?? NULL;
+            if (is_int($maxBytes)) {
+                Repository::$maxBytes = $maxBytes;
+            }
+
```

Comment:
not sure if there was existing test we could modify, liking setting max bytes very low see it fail and then set to large value and see it pass. To config the repo setting is respected

Reply:
Added a test.